component/src/main/java/com/celements/structEditor/fields/LabelTagPageType.java

```diff
@@ -32,7 +34,12 @@ protected String getViewTemplateName() {
   @Override
   public void collectAttributes(AttributeBuilder attrBuilder, DocumentReference cellDocRef) {
     attrBuilder.addCssClasses("structuredDataEditorLabel");
-    attrBuilder.addNonEmptyAttribute("for", attrBuilder.getAttribute("id").orElse(""));
+    XWikiObjectFetcher.on(modelAccess.getOrCreateDocument(cellDocRef))
+        .fetchField(CellClass.FIELD_ID_NAME)
+        .stream().findFirst().ifPresent(id -> {
+          attrBuilder.addId(id + "_Label");
```

Comment:
use a letter based prefix to ensure that a valid id is generated. An id must start with an letter. see: https://developer.mozilla.org/de/docs/Web/HTML/Globale_Attribute/id